Kitodo/src/main/resources/kitodo_metadataDisplayRules.xml

```diff
@@ -12,8 +12,8 @@
  *
 -->
 
-<!--<displayRules xmlns="http://meta.goobi.org/displayRules" xmlns:tns="http://meta.goobi.org/displayRules" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://meta.goobi.org/displayRules http://meta.goobi.org/displayRules.xsd">-->
-<displayRules xmlns="http://meta.goobi.org/displayRules" xmlns:tns="http://meta.goobi.org/displayRules">
+<!--<displayRules xmlns="http://meta.kitodo.org/displayRules" xmlns:tns="http://meta.kitodo.org/displayRules" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://meta.goobi.org/displayRules http://meta.goobi.org/displayRules.xsd">-->
+<displayRules xmlns="http://meta.kitodo.org/displayRules" xmlns:tns="http://meta.kitodo.org/displayRules">
```

Comment:
Be careful when renaming name spaces. It may add an additional layer of complexity when updating legacy installations. Changing the name space to one which is under our control is acceptable. We should keep in mind that it might turn out to be helpful to publish the semantics of the objects in the future, so the URL should be chosen carefuelly. We already introduced the domain base `http://names.kitodo.org/` for [GPath](https://github.com/kitodo/kitodo-production/blob/2.x/Goobi/src/org/kitodo/production/lugh/ld/GraphPath.java#L114-L137)s in `2.x`, [see discussion](https://github.com/kitodo/kitodo-production/pull/631#issuecomment-276664018). I would like to recommend to use the following name space for legacy display rule definitions:

    xmlns="http://names.kitodo.org/legacy/displayRules"